### DISCUSSION TITLE
Group By Repository

### DISCUSSION THREAD

**Discussion by andre-wells** [community/community#7512] [Projects and Issues] [answered]
For Projects (beta), we can group by any new field we add and a few default fields (such as `Status` and `Milestone`). 

One of the default fields we can include is `Repository`.  If the purpose of Projects (beta) is to give a single source of truth across multiple repositories, it seems strange that we can't group by  this field.

When dealing with several microservices in separate repositories, we have one or more team members contributing to each area. Presently I need to manually sort issues to retain some form of logical grouping with the Repository field visible.

**Comment by theilgaz** [comment]
This would be perfect to sort issues group by `Repository` order by `Priority`.

**Comment by andre-wells** [comment]
I recognize that using the current Group tooling lets us easily drag issues from one group into another.  This is impossible for grouping by Repository so maybe this can manifest as a "Read-only Group" of sorts?

**Comment by lysut** [comment, marked as answer]
Looks like this might be handled in [#283](https://github.com/github/roadmap/issues/283)

**Comment by azenMatt** [comment]
Hi everyone - this is now live :) 

you can check out the other latest updates here - https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/
